dash-pipeline/bmv2/dash_metadata.p4

```diff
@@ -187,6 +187,7 @@ struct metadata_t {
     bool dropped;
     encap_data_t encap_data;
     overlay_rewrite_data_t overlay_data;
+    bit<16> tunnel1_id;
```

Comment:
maybe we can simply name it as tunnel id. the reason is that with "direct" routing type, this tunnel will show up as underlay0 instead of underlay1.

Reply:
Renamed to dash_tunnel. Tunnel is taken by main SAI.